### DISCUSSION TITLE
Support Android OS

### DISCUSSION THREAD

**Discussion by tccloudz** [forcedotcom/cli#2242] [Ideas]
**Is your feature request related to a problem? Please describe.**
I use a terminal emulator app (Termux) on my Android phone. I wanted to use the cli on the phone via the emulator app. I was able to install the CLI using npm, however, whenever I try to run any commands an error is returned stating that the Android is an unsupported OS.

**What are you trying to do**
Be able to run the tool on my android phone as I perform a lot of tasks on the phone, including light development. 

**Describe the solution you'd like**
I am able to successfully install the CLI tool on the phone via the terminal emulator app, I do not see any reason why I am not able to run any of the commands.

**Describe alternatives you've considered**
As an alternative I was able to install the Force.com CLI successfully and be able to run the commands via that tool.

**Comment by mshanemc** [comment]
I'm guessing that if we unblocked the OS check, there'd be a bunch of OS-related things behind it that cause problems (fs structure and permissions, keychains, commands to open stuff in the browser etc).
